doc/quickstart/istsos_quickstart.rst

```diff
@@ -30,111 +33,78 @@ Open a terminal and execute the following commands:
 
-Press the "Coordinates system" button to manage the Coordinate Reference System (CRS).
-
-*Default EPSG code*: this will be the native Coordinate Reference System of your geometries in the database.
-
-*Permitted EPSG*: here you can configure which other Coordinate Reference Systems your system will support and reproject if requested.
-
-**6) Configure your getobservation request options**
-
-From the toolbar buttons menu press the "GetObservation Configuration" button to
+#. From the toolbar buttons menu, press the "GetObservation Configuration" button to
 set maximum time interval per getObservation request (zero for no limits) and aggregation no-data value.
 
-**7) Configure your service proxy address**
-
-
-The Proxy URL field is the base URL seen beyond a reverse proxy.
+#. Configure your service proxy address. The Proxy URL field is the base URL seen beyond a reverse proxy.
 
 
 Create a new service
-================================================================================
-
+====================
 
 Now that you have configured istSOS, it's time to create a new service instance.
 
-**1) Create a new service**
-
-
-From the toolbar buttons menu press the "new service" button:
-
-
-Choose the name of your instance. In this workshop use the name **demo**.
+#. From the toolbar buttons menu press the "new service" button and click :guilabel:`Next`.
 
+#. Type **demo** for the Service name.
 
-In the EPSG field you can set the database native projection. Leaving it empty,
-the default one will be used.
+#. In the EPSG field you can set the database native projection. Leave it empty to use the default. Click :guilabel:`Next`.
 
-Clicking on the "Customize database" box, you are able
-to change the default database configuration for
-this new istSOS instance.
-
-**2) Press “next”**
-
-As you pressed the “next” button, the server status panel is displayed.
-If something's gone wrong during the creation you will see here an alert.
+#. Clicking on the "Customize database" box, you are able to change the default database configuration for this new istSOS instance.
 
+#. Press :guilabel:`Next`. The server status panel is displayed. If something's gone wrong during the creation you will see an alert here.
 
 .. note::
     The istSOS “demo” instance has inherited all the configuration options from the default configuration. If, for any reason you decide to modify them, the changes will affect only this instance.
 
 
 Add a new procedure
-================================================================================
-
-From the “services” drop down button of the admin interface (http://localhost/istsos/admin ) choose the desired instance.
+===================
 
-From the toolbar buttons menu press the "New procedure" button.
+#. From the “services” drop down button of the admin interface (http://localhost/istsos/admin ) choose the desired instance.
```

Comment:
This should be auto-numbered, but it is showing the hash. @geographika can you help? I tried fixing the indent, etc